jina/drivers/index.py

```diff
@@ -39,8 +39,7 @@ class VectorIndexDriver(BaseIndexDriver):
     If `method` is not 'delete', documents without content are filtered out.
     """
 
-    @staticmethod
-    def _get_documents_embeddings(docs: 'DocumentSet'):
+    def _get_documents_embeddings(self, docs: 'DocumentSet'):
```

Comment:
Why not static anymore?

Reply:
because the inherited function needs instance attributes